README.md

```diff
@@ -47,6 +47,16 @@ In particular, bubblewrap uses `PR_SET_NO_NEW_PRIVS` to turn off
 setuid binaries, which is the [traditional way](https://en.wikipedia.org/wiki/Chroot#Limitations) to get out of things
 like chroots.
 
+Features
+--------
+
+Bubblewrap tries to expose most useful features that unprivileged user
```

Comment:
Make this more definitive.

Bubblewrap exposes useful features that unprivileged user